doc/release-notes.md

```diff
@@ -1,5 +1,107 @@
 # release notes
 ## current release
+### release-v17-20200908
+- release date: 2020-09-11
+- status: available
+- changes:
+  - pbta-histologies.tsv updates:
+    - clinical updates to columns: `race`, `ethnicity`, ETC.....
```

Comment:
```suggestion
    - clinical updates to columns: `race`, `ethnicity`, ETC.....
```
I put `ETC....` here as a placeholder 😃 . Can you add here the column names of the clinical data that was refreshed?